### PR TITLE
changed actions to fix the live refresh issue

### PR DESCRIPTION
**Pull request by kvhnuke**

### CODE REVIEW

File: apps/ethvm/src/states/actions.ts [hunk 1]
```diff
@@ -3,22 +3,22 @@ import sEvents from '@app/configs/socketEvents.json'
...
 }
 
-const socket_newTx = function({ commit }, tx: TxLayout | TxLayout[]) {
+const socket_socketNewtx = function ({ commit }, tx: TxLayout | TxLayout[]) {
   commit(sEvents.newTx, tx)
   this._vm.$eventHub.$emit(sEvents.newTx, Array.isArray(tx) ? new Tx(tx[0]) : new Tx(tx))
 }
 
-const socket_connect = ({}, tx: TxLayout) => {
+const socket_socketConnect = function ({}, tx: TxLayout) {
```
Comment (aldoborrero) [inline apps/ethvm/src/states/actions.ts]: Two related questions:

1) Is function needed? Or can we use shorter version with the arrow syntax?

2) Do we need to pass an object and a tx class as fn parameters? It's not being used, right?

Comment (kvhnuke) [reply apps/ethvm/src/states/actions.ts]: yea cz `this` wasnt working :(

Comment (aldoborrero) [reply apps/ethvm/src/states/actions.ts]: ouch! Then we need to add rules to avoid the linter complain!

Comment (kvhnuke) [reply apps/ethvm/src/states/actions.ts]: 2. first parameter is `{ commit, state, dispatch }` if we need any of them we can use it, second parameter is the payload

Comment (kvhnuke) [reply apps/ethvm/src/states/actions.ts]: we can make $eventHub part of the state, that will let us go back to arrow functions

Comment (aldoborrero) [reply apps/ethvm/src/states/actions.ts]: Ok got it! Then decide which strategy is better in this particular case (both will work) 👍

**Pull request merged**